packages/manager/apps/dedicated/client/app/components/dedicated-cloud/users/activeDirectories/add/federation-add.html

```diff
@@ -0,0 +1,238 @@
+                    data-ng-model="$ctrl.model.username"
+                    required
+                />
+            </oui-field>
+            <oui-field
+                class="col-md-6"
+                data-label="{{ 'dedicatedCloud_USER_AD_ADD_field_label_password' | translate }}*"
+                data-label-popover="{{ 'dedicatedCloud_USER_AD_ADD_field_tooltip_password' | translate }}"
+            >
+                <oui-password
+                    name="{{ :: $ctrl.FIELD_NAME.password }}"
+                    placeholder="{{ :: $ctrl.PLACEHOLDER.password }}"
+                    model="$ctrl.model.password"
+                    required
+                ></oui-password>
+            </oui-field>
+        </div>
+        <div class="col-sm-12">
+            <oui-field
+                class="col-md-6"
+                data-label="{{ 'dedicatedCloud_USER_AD_ADD_field_label_baseDnForUsers' | translate }}*"
+                data-label-popover="{{ 'dedicatedCloud_USER_AD_ADD_field_tooltip_baseDnForUsers' | translate }}"
+            >
+                <input
+                    class="oui-input"
+                    name="{{ :: $ctrl.FIELD_NAME.baseDnForUsers }}"
+                    placeholder="{{ :: $ctrl.PLACEHOLDER.baseDnForUsers }}"
+                    type="text"
+                    data-ng-model="$ctrl.model.baseDnForUsers"
+                    required
+                />
+            </oui-field>
+            <oui-field
+                class="col-md-6"
+                data-label="{{ 'dedicatedCloud_USER_AD_ADD_field_label_baseDnForGroups' | translate }}*"
+                data-label-popover="{{ 'dedicatedCloud_USER_AD_ADD_field_tooltip_baseDnForGroups' | translate }}"
+            >
+                <input
+                    class="oui-input"
+                    name="{{ :: $ctrl.FIELD_NAME.baseDnForGroups }}"
+                    placeholder="{{ :: $ctrl.PLACEHOLDER.baseDnForGroups }}"
+                    type="text"
+                    data-ng-model="$ctrl.model.baseDnForGroups"
+                    required
+                />
+            </oui-field>
+        </div>
+    </div>
+
+    <p data-translate="dedicatedCloud_USER_AD_ADD_mendatory_fields"></p>
+
+    <span data-translate="dedicatedCloud_USER_AD_ADD_foot_information"></span>
+
+    <a href="{{:: $ctrl.guideFederationLink }}" target="_blank" rel="noopener">
+        <span data-ng-bind=":: $ctrl.guideFederationLink"></span>
+        <span class="oui-icon oui-icon-external-link" aria-hidden="true"></span>
+    </a>
+
+    <div class="mt-4 col-sm-12">
+        <oui-form-actions
+            submit-text="{{ 'dedicatedCloud_USER_AD_ADD_submit' | translate }}"
+            on-submit="$ctrl.onSubmit()"
+            disabled="$ctrl.loaders.submitting || $ctrl.loaders.submitted || form.$invalid"
+        ></oui-form-actions>
+    </div>
+</form>
+
+<div
+    data-ng-if="$ctrl.loaders.submitted"
+    class="modal"
+    role="dialog"
+    style="display: block;"
```

Comment:
```suggestion
```

Reply:
It's applied I changed with d-block in class balise